site/en/docs/devtools/settings/preferences/index.md

```diff
@@ -0,0 +1,686 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Preferences"
+authors:
+  - sofiayem
+date: 2023-02-14
+#updated: YYYY-MM-DD
+description: "A comprehensive reference of DevTools preferences."
+---
+
+Configure the appearance and behavior of DevTools and its panels using the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings**](/docs/devtools/settings/#open) > **Preferences** tab.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/Ms79s5pwzGOXMtzdFYxm.png", alt="The Preferences tab.", width="800", height="582" %}
+
+To find out what each setting does, search this page for the setting's name and expand its description.
+
+To configure other aspects of DevTools, see [Other tab](/docs/devtools/settings/other).
+
+## Preferences
+
+The **Preferences** tab lists both general customization options and panel-specific ones.
+
+To set preferences, open {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [Settings](/docs/devtools/settings/#open) > **Preferences** and scroll down to one of the sections described next.
+
+To restore default preferences, scroll down to the end of the **Preferences** tab and click **Restore defaults and reload**.
+
+### Appearance
+
+This section lists options that customize DevTools appearance.
+
+{% Details %}
+
+{% DetailsSummary %}
+**Themes** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} sets a color theme for DevTools UI.
+{% endDetailsSummary %}
+
+<fieldset>
+   <legend>Themes:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/m0URI9jodUrMxioYEZ11.gif", alt="Changing DevTools theme from system preference to dark to light.", width="400", height="400", style="float:right;", class="screenshot" %}
+
+- System preference
+- Light
+- Dark
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+**Panel layout** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} arranges panes in panels.
+{% endDetailsSummary %}
+
+Affects **Elements** > **Styles** and sister tabs, and the **Sources** > **Debugger** pane. The **auto** option makes the layout depend on DevTools width.
+
+<fieldset>
+   <legend>Panel layout:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/RO1dKsa37xhMhqCKRD6j.gif", alt="Changing the Element's panel layout from horizontal to vertical.", width="400", height="400", style="float:right;", class="screenshot" %}
+
+- horizontal
+- vertical
+- auto
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+**Color format** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} sets the format of CSS color values in **Elements** > **Styles**.
```

Comment:
Take note that this is deprecate in M112

Reply:
Noted